client/packages/android/app/src/main/java/org/openmsupply/client/NativeApi.java

```diff
@@ -499,7 +510,7 @@ public FrontEndHost(JSObject data) {
             // so that we can display the local server IP for users to connect to the API
             if (data.getBool("isLocal") && (ip.equals("127.0.0.1") || ip.equals("localhost"))) {
```

Comment:
Sorry I missed this in the previous PR review. 

I was wondering at this stage, wouldn't ip address already be translated (since JSObject data comes from front end, and front end gets it from serviceInfoToObject). 

This is definitly not a PR request atm (don't want to re-test this bit atm), but just wondering, maybe we can simplify in the future.

Reply:
you'd think so, but there's also this:

```
export const DEFAULT_LOCAL_SERVER = {
  protocol: 'https' as 'https' | 'http',
  port: 8000,
  ip: '127.0.0.1',
  clientVersion: '',
  hardwareId: '',
  isLocal: true,
};
```

which gives this:

![Screenshot 2023-11-22 at 5 09 33 PM](https://github.com/msupply-foundation/open-msupply/assets/9192912/5aaa7e05-0b78-462c-b168-004e9a78a991)